src/integrationtests/resources/com/aws/greengrass/integrationtests/sync_map.yaml

```diff
@@ -0,0 +1,50 @@
+---
+services:
+  aws.greengrass.Nucleus:
+    configuration:
+      runWithDefault:
+        posixUser: nobody
+  aws.greengrass.ShadowManager:
+    configuration:
+      synchronize:
+        shadowDocuments:
+          Thing1:
+            classic: true
```

Comment:
Is `classic: true` still the default value?

Reply:
If there is a thing configuration object provided, and `classic` configuration is not provided, then the default value is `true`